examples/batch/main.go

```diff
@@ -20,6 +22,7 @@ func main() {
 
 	// create a batch
 	batch := db.NewBatch(rosedb.DefaultBatchOptions)
+	defer batch.Rollback()
```

Comment:
use will either call `Commit` or `Rollback`, so need to call this in defer

Reply:
done